specification/esql/_types/TableValues.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { double, integer, long } from '@_types/Numeric'
+
+/** @variants container */
+export interface TableValues {
```

Comment:
```suggestion
export interface TableValueContainer {
```

Just a naming suggestion to be consistent with the existing specification, LGTM otherwise!

Reply:
(or `TableValuesContainer` - not sure if singular or plural makes more sense here)

Reply:
I think plural does. I'll make the change.